src/TRestRawSignalRemoveChannelsProcess.cxx

```diff
@@ -143,24 +145,55 @@ void TRestRawSignalRemoveChannelsProcess::LoadConfig(const string& configFilenam
 ///
 TRestEvent* TRestRawSignalRemoveChannelsProcess::ProcessEvent(TRestEvent* inputEvent) {
     fInputSignalEvent = (TRestRawSignalEvent*)inputEvent;
+    fInputSignalEvent->InitializeReferences(GetRunInfo());
 
-    for (int n = 0; n < fInputSignalEvent->GetNumberOfSignals(); n++) {
-        TRestRawSignal* sgnl = fInputSignalEvent->GetSignal(n);
+    if (fReadoutMetadata == nullptr) {
+        fReadoutMetadata = fInputSignalEvent->GetReadoutMetadata();
+    }
 
-        Bool_t removeChannel = false;
-        for (unsigned int x = 0; x < fChannelIds.size() && !removeChannel; x++)
-            if (sgnl->GetID() == fChannelIds[x]) removeChannel = true;
+    if (fReadoutMetadata == nullptr) {
+        std::cerr << "TRestRawBaseLineCorrectionProcess::ProcessEvent: readout metadata is null" << std::endl;
```

Comment:
Since we typically import the namespace `std`, we can avoid explicitly typing it (e.g. `cerr` instead of `std::cerr`) for consistency.

It's not always a bad idea to explicitly type the namespace even thought it's already included, but I like to do this only when the symbol you are using it's not so frequently used that other users may not know which namespace it comes from.